src/mame/drivers/galaxian_rockclim.cpp

```diff
@@ -0,0 +1,294 @@
+// license:BSD-3-Clause
+// copyright-holders:David Haywood
+
+#include "emu.h"
+#include "includes/galaxian.h"
+
```

Comment:
You may as well throw the content in an anonymous namespace for the marginal improvement in link time.